### ISSUE TITLE
(test_hironx.py) the number of joints passed to `setJointAngles` might not match the model contained

### ISSUE THREAD

**Issue by 130s** [start-jsk/rtmros_hironx#6]
Continued from [here](https://github.com/start-jsk/rtmros_hironx/commit/756f0c1032e7f542981466adb3fe08f1a9c08001#commitcomment-5387144).

> Since, HIRONX model at JSK have 24 joints.

I see. But then, the released `Hironx` should be 15dof as far as I'm understood; on the terminal where simulation runs, I get the following when testcases that include `setJointAngles`.

```
virtual CORBA::Boolean SequencePlayerService_impl::setJointAngles(const OpenHRP::dSequence&, CORBA::Double) num of joint is differ, input:24, robot:15
```

**Comment by k-okada** [comment]
which modelfile are you using?
https://raw.github.com/start-jsk/rtmros_hironx/master/hironx_ros_bridge/models/kawada-hironx.dae

Also please check  why following jenkins jobs passed, if we need test code to check setJointAngles, please add them.
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/hironx-catkin/

**Comment by 130s** [comment]
I confirmed that running `test_hironx.py` (most likely on the ones that have `15-DOF`, not 23) causes something strange, as follows as I observed:
- An arm stop instantaneously, say less than a second, during a trajectory
- Arms make growling noise
- Calling `robot.servoOff/On` often don't return.
- With `robot.goOffpose` arms only reach the half way and servos turn off.

I'm so sure since this happened on multiple `Hiro` in different places. So, instead of investigating this case more on the real robot, first I'll take a closer look into the test file.
